### DISCUSSION TITLE
Cannot install via homebrew

### DISCUSSION THREAD

**Discussion by qthequartermasterman** [modular/modular#1209] [Q&A] [answered]
I seem to be unable to install the modular package via homebrew on an M1 Macbook Air due to a SHA256 mismatch with the downloaded file and the expected file. Is there any advice on how to overcome this issue?

I get the same error below when either using the default instructions or the manual instructions on the [download page](https://developer.modular.com/download).

```bash
$ brew update
Already up-to-date.
$ brew install modularml/packages/modular
==> Fetching modularml/packages/modular
==> Downloading https://dl.modular.com/public/installer/raw/names/modular-mac-arm64/versions/latest/modular-v0.2.2-macos-arm64.tar.gz
Already downloaded: /Users/username/Library/Caches/Homebrew/downloads/449bbcc6f922c91a915ece011f2f8c463f37911b139d29845835184a040e46c8--modular-v0.2.2-macos-arm64.tar.gz
Error: modular: SHA256 mismatch
Expected: 87d3fc5eaabce62e5f3cb07994cf767422fde261133a7b2c8d4c34b33ff42969
  Actual: 0da8501e277c5779a73d4a2a9f3ac7c65a7320d6ba602032610184a1566dcaaf
    File: /Users/username/Library/Caches/Homebrew/downloads/449bbcc6f922c91a915ece011f2f8c463f37911b139d29845835184a040e46c8--modular-v0.2.2-macos-arm64.tar.gz
To retry an incomplete download, remove the file above.
```

I am running `brew==4.1.18` (which appears to be the latest as of posting) on an M1 MacBook Air (2020) with Ventura 13.4.

**Comment by Sharktheone** [comment, marked as answer]
Hmm, this seems like you have some network problems. When I download the file with curl, i get the expected checksum. Maybe you can try downloading it via a VPN or restart your router etc.
